test/com/google/enterprise/adaptor/database/TupleReaderTest.java

```diff
@@ -118,6 +118,24 @@ public void testVARCHAR() throws Exception {
     assertEquals(golden, result);
   }
 
+  @Test
+  public void testNullVARCHAR() throws Exception {
```

Comment:
testVarchar_null, here and throughout, but don't change the existing test names.